bun/bun.go

```diff
@@ -497,3 +499,14 @@ func handleInfo(ex *exchange.Exchange) {
 		"hostname": hostname,
 	})
 }
+
+func handleEnv(ex *exchange.Exchange) {
+	ex.ResponseWriter.Header().Set("Content-Type", "application/json")
+	data := make(map[string]interface{})
+	for _, e := range os.Environ() {
+		pair := strings.SplitN(e, "=", 2)
+		data[pair[0]] = os.Getenv(pair[0])
```

Comment:
The [`strings.Cut`](https://pkg.go.dev/strings#Cut) would read better I think, no?

```suggestion
		name, value, _ := strings.Cut(e, "=")
		data[name] = value
```